src/multi_vector_simulator/E0_evaluation.py

```diff
@@ -64,19 +64,27 @@
 
 def evaluate_dict(dict_values, results_main, results_meta):
     """
+    Processes all simulation outputs by evaluating oemof results, asset capacities and dispatch as well as all KPIs.
 
     Parameters
     ----------
     dict_values: dict
         simulation parameters
+
     results_main: DataFrame
         oemof simulation results as output by processing.results()
+
     results_meta: DataFrame
         oemof simulation meta information as output by processing.meta_results()
 
     Returns
     -------
 
+    Notes
+    -----
+
+    Tested with:
+    -
```

Comment:
Is there a test that you can add here?

Reply:
Added: ab930bd6a462ae8a3d11e5c1d5a1cb5fec3fa992